README.md

```diff
@@ -1 +1,4 @@
-# kubecf-kubectl
\ No newline at end of file
+# kubectl for KubeCF
+
+This repository contains the Dockerfile for the kubecf-kubectl image.
+It contains only the kubectl binary in the final image.
```

Comment:
Please explain a bit _why_ we have this image.

Reply:
We don't need to explain why. This is an image containing kubectl to be used by KubeCF, this is it. _How_ it's used, and _why_ this repository exists is irrelevant here. Also, it's not the concern of this repository to state how it's used in KubeCF.